public/index.php

```diff
@@ -15,7 +15,9 @@
 use Symfony\Component\ErrorHandler\Debug;
 use Symfony\Component\HttpFoundation\Request;
 
-\defined('SULU_MAINTENANCE') || \define('SULU_MAINTENANCE', \getenv('SULU_MAINTENANCE') ?: false);
+if (!\defined('SULU_MAINTENANCE')) {
```

Comment:
Logical operators are for booleans only.

Reply:
I think at the end it results in the same and I don't see why it is important here to split this at current state into multiple lines. Would more discuss if envs are still the way here to go here.

Reply:
> why it is important here to split this at current state into multiple lines.

`||` is a operator for booleans. It is a _trick_ to use it this way.
Trick === Unsustainable.